benchmarks/README.md

```diff
@@ -4,6 +4,27 @@ The two main benchmarking scripts are
   - `experiment_runner.py` to run benchmark experiments, and
   - `result_analyzer.py` to aggregate the benchmark result in CSV form.
 
+## Prerequisites
+
+Follow the [contributing
+guide](https://github.com/pytorch/xla/blob/master/CONTRIBUTING.md) to set up
+the repo for development. Using the devcontainer is recommended.
+
+Clone the [`pytorch/benchmark`](https://github.com/pytorch/benchmark) repo into
```

Comment:
nit: I feel this line is not needed. Just in case, someone is using a different setup.

Reply:
I revised it to be more general.